ec2/tests/test_ec2.py

```diff
@@ -24,11 +24,14 @@
 from ec2 import connection
 from ec2 import instance
 from ec2 import utils
+from ec2 import floatingip
 from cloudify.mocks import MockCloudifyContext
+from cloudify.mocks import MockContext
 from cloudify.exceptions import NonRecoverableError
 
 TEST_AMI_IMAGE_ID = 'ami-e214778a'
 TEST_INSTANCE_TYPE = 't1.micro'
+FQDN = '((?:[a-z][a-z\\.\\d\\-]+)\\.(?:[a-z][a-z\\-]+))(?![\\w\\.])'
 
 
 class TestPlugin(testtools.TestCase):
```

Comment:
Created test_ec2_*module*.py for each module and moved the related functions in there.
**I also did this for the utils module even though currently all utils relate to a particular module.